src/signature.ts

```diff
@@ -1,5 +1,5 @@
-const secp256k1 = require('secp256k1')
-import BN = require('bn.js')
+import * as secp256k1 from 'secp256k1'
+import * as BN from 'bn.js'
 import { toBuffer, setLength, setLengthLeft, bufferToHex } from './bytes'
 import { keccak } from './hash'
 
```

Comment:
Did have a look at all module files, also checked the linking from `README`. This looks good and consistent now.